documentation/design/proposals/bridging_external.adoc

```diff
@@ -0,0 +1,80 @@
+== Overview
+
+A feature requested by several users is the ability to connect address spaces to external AMQP endpoints. NOTE: this proposal does not cover bridging between address spaces on the same cluster.
+
+The use cases covered by this proposal are:
+
+1. Forwarding messages from a local queue in a local address space to a destination on a remote AMQP endpoint
+1. Forwarding messages to a local queue in a local address space from a destination on a remote AMQP endpoint
+1. Sending messages to a remote AMQP endpoint via a local address space - by creating a connector and using prefixing
+1. Receiving messages from a remote AMQP endpoint via a local address space - by creating a connector and using prefixing
+
+== Design
+
+The address space specification would be expanded with 2 addition sub-objects, `connectors` and `forwarders`. A connector configures the infrastructure to connect to a remote AMQP endpoint and add address matching rules with the specified addresses. A forwarder specifies message routing rules to/from a local address to a remote address.  
+
```

Comment:
We have the capability to restrict outgoing connections via network policies today. Is that sufficient? Restricting which addresses to forward doesn't feel right to put in the infra configs somehow, since it does not have that knowledge.